packages/component/src/SendBox/SuggestedActions.js

```diff
@@ -48,8 +48,11 @@ export default connectSuggestedActions(
             buttonText={
               type === 'messageBack' ?
                 title || displayText
-              :
-                title || (typeof value !== 'string') ? JSON.stringify(value) : value
+              : title ?
+                  title
+                : (typeof value === 'string') ?
```

Comment:
The indentation looks a bit weird to me.

Reply:
Not only that, but a nested ternary is usually avoided. Maybe this can be extracted into a utility function and re-written as an if / else